corehq/apps/translations/utils.py

```diff
@@ -10,3 +15,20 @@ def get_file_content_from_workbook(wb):
         f.seek(0)
         content = f.read()
     return content
+
+
+def update_app_translations_from_trans_dict(app, trans_dict):
+    if isinstance(app, LinkedApplication):
+        if app.add_ons.get('partial_commcare_translations') is True:
```

Comment:
The `show` command in `add_ons` was the closest analog I can find to `toggle_enabled` but that requires a `request` object, and this happens in an async celery task.